exporter/awsemfexporter/datapoint.go

```diff
@@ -12,20 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/*
+datapoint.go - calculate delta values or keep the value  for each OTEL type (e.g calculate delta for Counter)
+*/
```

Comment:
```suggestion
```

This isn't a package-level godoc string.  Descriptions of functionality like this should go on types or methods.

Reply:
I usually do both but I'm fine with removing it too. Done and thanks Anthony !!!